repl/src/main/scala/ammonite/repl/Util.scala

```diff
@@ -324,6 +328,7 @@ object Util{
 object Timer{
   var current = 0L
   var show = false
+  var startTime = System.nanoTime()
```

Comment:
This I was using for testing, if someone wants to check time elapsed between two points in program flow, one can set the `startTime` and print by subtracting from the `System.Nanotime` at the end. I will remove this if you think it won't be necessary in future otherwise I will put a comment explaining its  use